src/Azure.Functions.Cli/Actions/LocalActions/InitAction.cs

```diff
@@ -527,5 +530,13 @@ private static string AddLocalSetting(string localSettingsContent, string key, s
 
             return JsonConvert.SerializeObject(localSettingsObj, Formatting.Indented);
         }
+
+        public async Task FetchPackages(WorkerRuntime workerRuntime, ProgrammingModel programmingModel)
+        {
+            if (workerRuntime == Helpers.WorkerRuntime.node && programmingModel == Common.ProgrammingModel.V4)
+            {
+                await NpmHelper.Install();
```

Comment:
We're doing this to help the user, but they can easily do it on their own. For that reason, I'd prefer if we wrap this in a try/catch and print a warning instead of an error. Maybe something like this:
> Warning: You must run "npm install" manually